.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  building_release:
+    strategy:
+      fail-fast: true
+      matrix:
+        otp_release: ["OTP-24.0"]
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - name: Install Erlang/OTP
+        run: |
+          cd $RUNNER_TEMP
+          wget -O otp.tar.gz https://repo.hex.pm/builds/otp/ubuntu-18.04/${{ matrix.otp_release }}.tar.gz
+          mkdir -p otp
+          tar zxf otp.tar.gz -C otp --strip-components=1
+          otp/Install -minimal $(pwd)/otp
+          echo "$(pwd)/otp/bin" >> $GITHUB_PATH
+      - name: Build Elixir Release
+        run: |
+          make Precompiled.zip
+          mv Precompiled-v1.14.0-dev.zip elixir-${{ github.ref_name }}-${{ matrix.otp_release }}.zip
```

Comment:
Still need to keep to `v1.14.0-dev` for testing purpose. I think we can renaming it to `github.ref_name` later.

Reply:
you can also read the `VERSION` file:

```bash
$ vsn=${cat VERSION}
$ mv Precompiled-v${vsn}.zip elixir-${vsn}-${{ matrix.otp_release }}.zip
```